src/checks/geometry.py

```diff
@@ -0,0 +1,73 @@
+from geopandas import GeoDataFrame
+
+from src.config import CheckReturnList
+from src.utils import get_epsg_ease
+
+EPSG_WGS84 = 4326
+METERS_PER_KM = 1_000_000
+PRECISION = 6
```

Comment:
These are probably global constants rather than module-level constants, so it might be more logical for them to live in `config.py`. 

Also, `PRECISION` is a generic name that could mean anything, suggest `DECIMAL_PRECISION` as a less ambiguous name 🙂